Data/ICAO/ICAO_Airlines.txt

```diff
@@ -355,7 +355,7 @@ AOU	Air Tractor - Croatia	AIR TRACTOR
 AOV	Aero Vision - France	AEROVISION
 AOW	Air Omega Luftverkehrsgesellschaft mbH - Germany	WARLOCK
 AOX	Aerotaxi Del Valle - Colombia	AEROVALLE
-APA	Air Pacifica Airlines - Virgin Islands (US)	PACIFICA
+APA	Air Pacifica Airlines - Virgin Islands (US)	AIRWAYS
```

Comment:
Should be changed to `APA Airway Philippines Virtual Airline - Philippines AIRWAY`

Reply:
I'll move it over to the virtual airline callsign file then